app/views/content_items/document_collection.html.erb

```diff
@@ -22,7 +22,7 @@
 </div>
 
 <%= render 'shared/publisher_metadata_with_logo' %>
-<%= render 'shared/single_page_notification_button', content_item: @content_item %>
+<%= render 'shared/single_page_notification_button', content_item: @content_item, skip_account: "true" %>
```

Comment:
Could you add the info you put into the PR description about what skipping the account actually does into this PR? I think it would help, it wasn't obvious to me but your explanation was v useful.

Reply:
Yes good suggestion thank you, will do that 👍